### PR TITLE
Finish the whole app

### PR DESCRIPTION
**Pull request by heiheychan**

### CODE REVIEW

**Comment by h8rry** [comment]
You shouldn't need to run `rake asset:precompile`. If you do need to run it, you should remove the files.

File: app/assets/javascripts/angualr/controllers/people.js.erb [hunk 1]
```diff
@@ -0,0 +1,30 @@
+$(document).ready(function(){
```
Comment (f3r) [inline app/assets/javascripts/angualr/controllers/people.js.erb]: you have a typo in the directory name: `angualr` vs `angular`

File: app/assets/stylesheets/custom_css/event.css.scss [hunk 2]
```diff
@@ -0,0 +1,231 @@
...
+
+      li a:first-child {
+        border-top: none;
+      }
+    }
+  }
+
+  .main_cover {
+    height: 500px;
+    background: url('https://theadventuresofwhuffler.files.wordpress.com/2011/06/hong-kong-skyline1.jpg');
```
Comment (f3r) [inline app/assets/stylesheets/custom_css/event.css.scss]: you should include images from your assets folder, not directly from the internet

File: app/assets/stylesheets/custom_css/event.css.scss [hunk 3]
```diff
@@ -0,0 +1,231 @@
...
+          vertical-align: middle;
+        }
+
+        td {
+          font-size: 20px;
+        }
+      }
+    }
+  }
+
```
Comment (f3r) [inline app/assets/stylesheets/custom_css/event.css.scss:186]: no extra lines pls! :cry:

File: app/assets/templates/event/index.html [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        <tr>
+          <td>New Student Reception (in HK)</td>
+          <td>JULY</td>
+        </tr>
+      </table>
+    </div>
+
+  </div>
+</div>
+
```
Comment (f3r) [inline app/assets/templates/event/index.html:120]: no extra lines pls

File: app/assets/templates/people/index.html [hunk 5]
```diff
@@ -0,0 +1,191 @@
...
+          <div id="2010-2011" class="one_previous_pic col-lg-offset-2 col-lg-8" style="background: url('http://umich.edu/~hksa/1011/group.JPG'); background-size: cover;"></div>
+          <p class="one_previous_name col-lg-12">From left to right: Homan Tang, Raphael Lam, Kelvin Yuen, Euzane Lee, Kristie Wong, Keegan Ng, Keith Fong, Aaron Yan, Brian Lee</p>
+        </div>
+        <!-- Other previous board members will be added in the future -->
+      </div>
+    </div>
+    <p class="testing"></p>
+  </div>
+</div>
+
```
Comment (f3r) [inline app/assets/templates/people/index.html:173]: no extra lines pls

File: app/assets/templates/static_pages/index.html [hunk 6]
```diff
@@ -25,25 +35,55 @@
...
+        <a class="col-lg-6" target="_blank" href="https://www.facebook.com/hksa.umich?fref=ts&ref=br_tf"><div class="facebook fa fa-facebook"></div></a>
+      </div>
+      <div class="clear">
+      </div>
     </div>
 
     <!-- Footer -->
-    <div class="footer">
+    <div class="footer col-lg-12">
+      <p class="col-lg-offset-3 col-lg-6">Made by <a target="_blank" href="https://hk.linkedin.com/in/billchan1"><span>Bill Chan</span></a> with Love. <del>Sorry the site is not quite responsive.</del> It's responsive now. Yay!</p>
```
Comment (f3r) [inline app/assets/templates/static_pages/index.html:86]: don't be `Sorry the site is not quite responsive`, make it responsive sir!!

File: app/assets/stylesheets/custom_css/event.css.scss [hunk 7]
```diff
@@ -0,0 +1,231 @@
+.event {
```
Comment (f3r) [inline app/assets/stylesheets/custom_css/event.css.scss:1]: I see a lot of duplication in the CSS, you scope it page with a class at the top, but perhaps you should leave common classes outside that scope and reuse them

File: app/assets/javascripts/application.js [hunk 8]
```diff
@@ -12,6 +12,8 @@
 //
 //= require angular/angular
 //= require angular-route/angular-route
+//= require ng-token-auth/dist/ng-token-auth
```
Comment (f3r) [inline app/assets/javascripts/application.js:15]: You remove the authentication from the backend but you still left it in the frontend?

File: Gemfile [hunk 9]
```diff
@@ -10,15 +10,14 @@ gem 'unicorn'
 gem 'bower'
 # gem 'bcrypt', '~> 3.1.7'
 # gem 'active_hash'
-# Devise token
-gem 'devise_token_auth'
```
Comment (f3r) [inline Gemfile:14]: why?

File: app/assets/javascripts/angualr/controllers/people.js.erb [hunk 10]
```diff
@@ -0,0 +1,30 @@
...
+  });
+
+  $(document).on('click', '.2010-2011', function(){   
+    $("html, body").animate({             
+      scrollTop: $("#2010-2011").offset().top - 150
+    }, 1000);     
+  });
+})
+
+app.controller('peopleCtrl', ['$scope', function($scope){
```
Comment (f3r) [inline app/assets/javascripts/angualr/controllers/people.js.erb]: very skinny controllers in angular no? hehehe come on bill, add a bit of functionality in angular too k?

**Comment by heiheychan** [comment]
Thank you for reviewing my code! I will try to add more functionality on Wed. Got a lot of hw to do today! Have a nice day! :))

**Comment by heiheychan** [comment]
merged

**Pull request merged**